.github/workflows/static-check-docs.yaml

```diff
@@ -0,0 +1,68 @@
+on:
+  schedule:
+    - cron: '0 23 * * 1'
+
+name: Static check docs
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.15.x, 1.16.x]
```

Comment:
why two go versions when checking docs?

Reply:
The original script is a copy of static-checks.sh, so there are two go versions. I will change it to keep only 1.16.x